### PR TITLE
[cling-cpt] More splitting lines for getting rid of E501 error cont. [skip-ci]

### PR DESCRIPTION
**Pull request by saisoma123**
# This Pull request: Makes the code more flake8 compliant.

## Checklist:

- [X] tested changes locally
- [NA] updated the docs (if necessary)

This PR fixes issue #406 mentioned in (https://github.com/root-project/cling/issues/406)

### CODE REVIEW

File: interpreter/cling/tools/packaging/cpt.py [hunk 1]
```diff
@@ -97,10 +97,12 @@ def travis_fold_end(tag):
 
 
 def box_draw_header():
-    msg = 'cling (' + platform.machine() + ')' + formatdate(time.time(), tzinfo())
+    msg = 'cling (' + platform.machine() + ')'
+    + formatdate(time.time(), tzinfo())
```
Comment (jalopezg-r00t) [inline interpreter/cling/tools/packaging/cpt.py:101]: I doubt that breaking the line there works (syntax error in line 101).  Perhaps a missing `\`? (same below)

**Review by jalopezg-r00t — CHANGES_REQUESTED** [review]

File: interpreter/cling/tools/packaging/cpt.py (hunk 1 above)
Comment (saisoma123) [reply interpreter/cling/tools/packaging/cpt.py:101]: Yeah you're right I'll fix that thank you

**Comment by vgvassilev** [comment]
Can you merge this PR and #11010?

**Comment by saisoma123** [comment]
@vgvassilev Yeah just merged and rebased

**Review by vgvassilev — APPROVED** [review]

**Pull request merged**